### ISSUE TITLE
NVDA not reading any menu items in Edge Download Manager "More Options" menu

### ISSUE THREAD

**Issue by britechguy** [nvaccess/nvda#15453]
### Steps to reproduce:
Open MS-Edge (latest released version)
Bring up Downloads Manager/All Downloads List using CTRL + J or the Edge menu
Hit SHIFT + Tab to throw focus to the manager menu bar then right arrow twice to get to the More Menu Options Button
Expand that button
Use down arrow to go through the menu options (there are only 4, and when initially opened focus is on the first.  There is a separator between menu items 1 and 2 as well as 3 and 4).  You will hear nothing.

### Actual behavior:
Described in steps to reproduce

### Expected behavior:
Would expect to hear the menu items in this order:
1. Open Downloads Page
2. Clear All Download History
3. Download Settings
4. Show Downloads Button on Toolbar.

I do not believe that NVDA has ever announced separators, but if it is supposed to then those should be announced between menu items 1 and 2 as well as 3 and 4.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.2
#### Windows version:
Windows 11 Version 22H2
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge Version 116.0.1938.76 (Official build) (64-bit)
#### Other information about your system:
LG Gram 15 with 12th gen i5 processor
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Did not test
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Did not test
Narrator is announcing everything correctly, and NVDA is not misbehaving elsewhere, even in Edge itself.

**Comment by ruifontes** [comment]
With:
- Microsoft Windows	Windows 11 22H2 (10.0.22621) workstation AMD64	[64 bit]
- Msedge (Microsoft Edge)	116.0.1938.81	[64 bit]
and
- NVDA	alpha-29194,ec914517	[32 bit]
all menu options are read as they should be.

**Comment by seanbudd** [comment]
Closing as duplicate of the fixed #14916
